src/main/java/org/folio/dcb/repository/TransactionsRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.dcb.repository;
+
+import org.folio.dcb.domain.entity.Transactions;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface TransactionsRepository extends JpaRepository<Transactions, UUID> {
```

Comment:
Please check transaction_id format in schema . somewhere it is of type String and at some places along with type it has format also. This will lead to error.

Reply:
checked. 
id UUID PRIMARY KEY.

won't cause the error.